src/main/views/create-media-account.njk

```diff
@@ -85,17 +83,17 @@
     {% set fileInput = govukFileUpload({
         id: "file-upload",
         name: "file-upload",
-        label: {
-            text: photoType
-        },
         errorMessage: {
             text: formErrors.fileUploadError.message
+        },
+        attributes: {
+          'aria-labelledby': 'photoType'
         }
     }) if formErrors.fileUploadError.message else govukFileUpload({
         id: "file-upload",
         name: "file-upload",
-        label: {
-            text: photoType
+        attributes: {
+          'aria-label': 'Upload a file'
```

Comment:
needs to be able to be tranlsated, to the locales!

Reply:
Whoops, Done!